pkgs/development/compilers/picat/default.nix

```
@@ -1,27 +1,29 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, zlib, gcc7Stdenv }:
 
-stdenv.mkDerivation {
-  name = "picat-1.9-4";
+gcc7Stdenv.mkDerivation {
```

please move this to all-packages.nix, and just do:
```
  package = callPackage ../../../.nix {
    stdenv = gcc7Stdenv;
  };
```
```suggestion
stdenv.mkDerivation {
```